plugins/janus_videoroom.c

```diff
@@ -3349,6 +3362,28 @@ static void *janus_videoroom_handler(void *data) {
 						}
 					}
 				}
+				if(record_pause) {
```

Comment:
This is important, because you may set a recording to paused and then start it, and the fact that both succeeded may make you think that you started the recording as paused, and you can resume it later, which is not the case.